### ISSUE TITLE
generate weightMaps?

### ISSUE THREAD

**Issue by lazharkhelifi** [Kamnitsask/deepmedic#133]
Hi Kamnitsask,

I have used your code with the following config files:   deepmedic/examples/configFiles/

I have trained DeepMedic on the training set of Brats 2018 without **weight-maps** (as it is commented in the trainConfig.cfg file):
#weightedMapsForSamplingEachCategoryTrain = ["./weightMapsForeground.cfg","./weightMapsBackground.cfg"]

The test on the validation set of the Brats 2018, gives the following result:
ET: 0.73      WT: 0.75     TC: 0.74

1/ Have I missed something?
2/ Is it important to use **weight-maps**?
3/ How I can generate weight-maps for each category?

Thanks,
Laz

**Comment by Kamnitsask** [comment]
Hello there,

Weight maps are not used at all. They are left there in case I or any user wants to make any specific experiment with them. But I find that they confuse people. Probably I should hide this option...

Nope, your results are very bad, far from what is expected on this database.

I would guess that this big a problem can only be because you miss something very important:
1. Do you train on all/enough data correctly? Be sure that the model is still not only learning only from the 2 samples given in my example.
2. Did you create and supply brainmasks?
3. Did you normalize intensities as done in our papers / suggested in the documentation?

K
